packages/cli/src/index.ts

```diff
@@ -11,7 +11,11 @@ import { introspectionFromSchema } from 'graphql';
 import { printSchemaWithDirectives } from '@graphql-tools/utils';
 export { generateSdk, serveMesh };
 
-const { writeFile } = fsPromises || {};
+const { writeFile: _writeFile, mkdir: _mkdir } = fsPromises || {};
+const writeFile: typeof _writeFile = async (path, ...args) => {
+  await _mkdir(path, { recursive: true });
```

Comment:
Are you sure this works if the directory is already available?

Reply:
Yep, that's what `recursive` option do.

https://nodejs.org/api/fs.html#fs_fspromises_mkdir_path_options

Reply:
Great 👍 just wanted to make sure

Reply:
Could you add a changeset with "yarn changeset" and we can merge and release?